po/pl.po

```diff
@@ -717,6 +717,24 @@ msgstr "Odnowienie adresów dynamicznych"
 msgid "Authentication is required to renew dynamic addresses."
 msgstr "Wymagane jest uwierzytelnienie, aby odnowić adresy dynamiczne."
 
+#: src/network/org.freedesktop.network1.policy:154
+msgid "Reload network settings"
+msgstr "Ponowne wczytanie ustawień sieci"
+
+#: src/network/org.freedesktop.network1.policy:155
+msgid "Authentication is required to reload network settings."
+msgstr "Wymagane jest uwierzytelnienie, aby ponownie wczytać ustawienia sieci."
+
+#: src/network/org.freedesktop.network1.policy:165
+msgid "Reconfigure network interface"
+msgstr "Ponowna konfiguracja interfejsu sieciowego"
+
+#: src/network/org.freedesktop.network1.policy:166
+msgid "Authentication is required to reconfigure network interface."
+msgstr ""
+"Wymagane jest uwierzytelnienie, aby ponownie skonfigurować interfejs "
```

Comment:
ponownie skonfigurować → zrekonfigurować?

Reply:
It’s not a real word, I’m afraid. :)

Thanks for the merge!